spec/config_spec.rb

```diff
@@ -27,4 +33,63 @@ module Builderator
       expect(Config.build_name).to eq 'builderator'
     end
   end
+
+  RSpec.describe Config, '#compile' do
+    before(:example) do
+      Builderator::Config::GLOBAL_DEFAULTS.reset!
+      Builderator::Config.reset!
+    end
+
+    10.times do |itr|
```

Comment:
Given we're resetting the `Builderator::Config` before each test, are the `10.times` loops on these tests still needed?